webapp/src/main/java/org/opentestsystem/rdw/admin/repository/SchoolRepository.java

```diff
@@ -17,7 +17,7 @@
      * @param permissionScope the scope of access
      * @return all of the schools accessible through the given permission scope
      */
-    List<Organization> findAll(@NotNull PermissionScope permissionScope);
+    List<School> findAll(@NotNull PermissionScope permissionScope);
```

Comment:
Since I needed the school.natural_id column, I had to extend Organization in order to retrieve it from the DB.